installation/Makefile

```diff
@@ -26,7 +26,7 @@ resources:
 	-$(DOCKER_COMPOSE) -f local/docker-compose.yaml up ${RESOURCES} -d
 	-minikube delete
 	-. ./local/.env && minikube start \
-	           --feature-gates="TTLAfterFinished=true" \
+			   --feature-gates="TTLAfterFinished=true" \
 		   --insecure-registry=$${HOST_REGISTRY} \
 		   --cpus="2" \
                    --memory="2500mb" \
```

Comment:
Everything about the makefile don't upload in the PR, because is own your configuration